frontend/src/reviews/ReviewDecisionDisplay.tsx

```diff
@@ -63,13 +65,15 @@ export default function ReviewDecisionDisplay({ response, modelId }: ReviewDecis
                   </Typography>
                 )}
               </Stack>
-              <Typography variant='caption'>as {getRoleDisplay(response.role, modelRoles)}</Typography>
+              <Typography variant='caption'>
+                as {getRoleDisplay(response.role || 'Error fetching model role', modelRoles)}
```

Comment:
This is an unusual way for us to handle errors. Why does the "get response" endpoint not error in this scenario? If it did then we could handle the error as we normally would.

Reply:
It's not a problem getting the response object but getting the role. In our typing `role` is optional because regular comments do not have roles, only review responses.

Reply:
In theory the `role` property will always be there so we _could_ do `getRoleDisplay(response.role as string, modelRoles)` but I'm not keen on this approach?

Reply:
I see. I think the best approach is to do the following then:
```
{response.role && (
    <Typography variant='caption'>as {getRoleDisplay(response.role, modelRoles)}</Typography>
)}
```